### PR TITLE
Change argument parsing from getopt to argparse

### PR DESCRIPTION
**Pull request by JustinWonjaePark**
## Description
Changed argument parsing for CLI from getopt to argparse

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_source/cli.py [hunk 1]
```diff
@@ -21,6 +20,8 @@
 from .run_scanoss import get_scanoss_extra_info
 import yaml
```
Comment (soimkim) [inline src/fosslight_source/cli.py:22]: Remove this empty row.

**Review by soimkim — CHANGES_REQUESTED** [review]

File: src/fosslight_source/cli.py [hunk 2]
```diff
@@ -53,29 +53,35 @@ def main():
...
+    parser.add_argument('--help', '-h', help='Print help message', action='store_true', dest='help')
+    parser.add_argument('--version', '-v', help='Print FOSSLight Source Scanner version', action='store_true', dest='version')
+    parser.add_argument('--path', '-p', help='Path to analyze', dest='path', default="")
+    parser.add_argument('--json', '-j', help='Generate raw result in json format', action='store_true', dest='json')
+    parser.add_argument('--output', '-o', help='Output path', dest='output', default="")
+    parser.add_argument('--matched', '-m', help='Print additional information', action='store_true', dest='matched')
+    parser.add_argument('--format', '-f', help='Output file format (excel, csv, opossum, yaml)', dest='format', default="")
+    parser.add_argument('--scanner', '-s', help='Select which scanner to be run (scancode, scanoss, all)',
+                        dest='scanner', default="")
+    parser.add_argument('--timeout', '-t', help='Stop scancode scanning in given seconds', dest='timeout', default=0)
```
Comment (soimkim) [inline src/fosslight_source/cli.py:66]: Plz check timeout default value.

Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py]: Fixed.

File: src/fosslight_source/cli.py (hunk 1 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py]: Fixed.

**Pull request closed without merging**